syntax = "proto3";

package master_map;

service map{
  rpc inputSplits(input_split) returns(input_response) {}
  rpc reducer_inputs(input_split) returns(input_response) {}
}

message input_split {
  repeated string input= 1;
}

message input_response {
  string response = 1;
}